README.md

```diff
@@ -1,8 +1,8 @@
 [![Build Status](https://travis-ci.org/SineQuaNonSoftware/puppet-proxmox.svg?branch=master)](https://travis-ci.org/SineQuaNonSoftware/puppet-proxmox)
 
-# proxmox
+# puppet-proxmox
```

Comment:
Most modules Drop the puppet in their homepage and only use it in the repo name. Imagine if all modules started with puppet. Yes, I'm on the puppet forge, I know this is for puppet...